drivers/hmis/app/graphql/types/application/user.rb

```diff
@@ -18,7 +18,7 @@ class Application::User < Types::BaseObject
     field :date_deleted, GraphQL::Types::ISO8601DateTime, null: true
 
     def name
-      [object.user_first_name, object.user_last_name].compact.join(' ')
+      [object.first_name, object.last_name].compact.join(' ')
```

Comment:
I don't know if this was working before, but `user_first_name` and `user_last_name` don't seem to exist on `Hmis::User`, which is why I made this change